core-plugins/src/main/java/co/cask/hydrator/plugin/transform/LogParserTransform.java

```diff
@@ -198,6 +205,17 @@ private String getLog(StructuredRecord input) {
     }
   }
 
+  private boolean validateInputSchemaType(Schema.Type inputSchemaType) {
```

Comment:
should change this to throw an exception. The caller can either choose to catch it and log it (in getLog), or let it go up (configurePipeline)